src/main/java/com/ecwid/maleorang/MailchimpObject.kt

```diff
@@ -19,6 +20,10 @@ open class MailchimpObject {
     @JvmField
     val mapping: MutableMap<String, Any?> = MailchimpObjectMapping(this)
 
+    @JvmField
+    @Field
+    var _links: List<String>? = null
```

Comment:
No, I don't think this is a good idea to add this field to the base class, since the class is used also for requests, where there is no such fields.

I personally think that _links fields are useless, don't they? I'd prefer them not be added at all. But if you want to add them, please add them to specific response classes.